src/Sulu/Bundle/MediaBundle/Resources/public/js/services/user-settings-manager.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(function() {
+
+        'use strict';
+
+        var instance = null,
+            mediaLanguageKey = 'mediaLanguage',
+            mediaListViewKey = 'collectionEditListView',
+            lastVisitedCollectionKey = 'last-visited-collection';
+
+        /** @constructor **/
+        function UserSettingsManager() {
+        }
+
+        UserSettingsManager.prototype = {
+            getMediaLocale: function() {
+                return app.sandbox.sulu.getUserSetting(mediaLanguageKey) || this.sandbox.sulu.user.locale;
```

Comment:
please use 'Husky' instead of 'app'. We try to get rid of the 'app' and 'App' constants